packages/corvid-fake-local-mode-editor/src/editor.js

```diff
@@ -139,6 +140,10 @@ const loadEditor = async (
     sendRequest(socket, "UPDATE_DOCUMENT", editorState.siteDocument);
 
   const saveCodeFiles = async () => {
+    if (extraArgs.failOnClone === "true") {
+      // eslint-disable-next-line no-console
+      console.log("[CORVID] failOnClone is set");
```

Comment:
it is really weird that any message starting with [CORVID] means a fatal error
please open a ticket to change that

Reply:
done